content/case_histories/hoop_region_norway/processinginterpretation.rst

```diff
@@ -0,0 +1,210 @@
+
+
+A qualitative interpretation of the CSEM inversion results supports the outcome of the Alternative, Central and Bjaaland wells. A prominent resistivity anomaly is recovered at Central, in which there was a significant oil discovery, which is in agreement with the high resistivity values measured at the reservoir location. On the other hand, the Realgrunnen structures penetrated at Alternative and Bjaaland, two dry wells, are related to low resistivity values that support the petrophysical outcome.
+At the location of the Hanssen discovery well, a subtle high resistivity anomaly is observed. The relatively small magnitude of this feature is the product of a 3D effect in the CSEM data, a consequence of the location of the CSEM line with respect to the location and the size of the reservoir (see :numref:`fig_geology_region_map_setup` b).   Although in many circumstances the 2.5D approximation is a good one, if the resistive reservoir is relatively small, or the 2D line being inverted is close to the edge of the reservoir, then the effect of the of the reservoir on the CSEM data is diminished, and consequently the resistivity recovered by a 2.5D inversion is reduced from its true value.  Loseth et al. illustrate this with a synthetic modelling study in which they demonstrate the reduction in inverted resistivity with width of the target reservoir and proximity of the CSEM line to the reservoir edge. In the case of Hanssen, the reservoir is both relatively small and the line is close to the edge of the body, leading to the lower resistivity at this location.  The effect of this on the quantitative interpretation is discussed in later sections.
+
+
+Seismic Data
+------------
+
+The ultimate quality of any inversion depends on the quality of the input data, and to ensure a robust result, data must be optimally conditioned for the workflow to be applied. In this example, the input seismic offset gathers were pre-conditioned prior to inversion to increase the overall signal-to-noise ratio and avoid any remaining noise having a high impact on the inversion results. The pre-conditioning also corrected for offset dependent frequency loss and ensured that the gathers were aligned. Since data pre-conditioning, if not applied carefully, could significantly affect AVO behavior of target reflectors, the sensitivity of inversion results to the pre-conditioning approaches applied has been thoroughly tested and analyzed. It is important to note that all conditioning steps were parameterized keeping in mind the need to preserve amplitude behavior, and the minimum possible pre-conditioning was undertaken. The processes applied are summarized in Singleton (2009).
```

Comment:
Some of the references don't exist in the material provided. Without knowing the title of the paper, I would be speculating. I will make a list of all missing references.